Add documentation for series copy, optional series and draft courses (#598)
dodona-edu/dodona-edu.github.io: en/guides/teachers/creating-a-course/index.md

Reviewer: Unrelated to the ticket, but now that we're touching it: do we want to mention the new "LLM questions enabled" setting?

* `Teacher`: the name or names of the instructors of the course.

> * `Draft course`: whether the course is a draft course or not. Draft courses can be used to prepare courses before they are available to students. It can also be used to use the course as a template or question bank to copy content from other courses. A draft course can't have an academic year, and the question and access rights options also can't be changed.

Reviewer: ```suggestion
* `Draft course`: whether the course is a draft course or not. Draft courses can be used to prepare courses before they are available to students. A draft course can also be used as a template or question bank to copy content from, into other courses. A draft course can't have an academic year, and the question and access rights options also can't be changed.
```

> * `Draft course`: whether the course is a draft course or not. Draft courses can be used to prepare courses before they are available to students. A draft course can also be used as a template or question bank to copy content from, into other courses. A draft course can't have an academic year, and the question and access rights options also can't be changed.

Reviewer: Do we want to mention here that students cannot see draft courses?

Reviewer: Reading this again, the double use of "question" (question bank & the question later) is a bit confusing since they mean different things.

```suggestion
* `Draft course`: whether the course is a draft. Draft courses let you prepare a course before it's available to students. A draft course can also serve as a template or content bank to copy exercises and reading activities into other courses. It can't have an academic year, and its `Questions` and access-rights options can't be changed either.
```

Also seems like we don't describe the `Questions` flag here in this list (unrelated to the scope of this PR)

* `Academic year`: the academic year in which the course is offered.

* `Language`: the natural language in which the course is given. This can be set to any of the supported languages (e.g. Dutch, English, French, ...). It is optional and helps users find courses in their preferred language. Additionally, the course language determines in which language the activities in the course are shown: if an activity has a description available in the course language, that description will be used.

* `Programming language`: the programming language that is used in the course. This is optional and only used to help users find courses in their preferred programming language.

* `Description`: an optional piece that is displayed at the top of the course page. For formatting the description, you can use [Markdown](https://en.wikipedia.org/wiki/Markdown). Dodona uses [kramdown](https://kramdown.gettalong.org) for rendering Markdown, which supports many extensions of the standard Markdown. For more explanation on how this formatting works, you can go [here](/en/references/exercise-description/).

> * `Questions`: these settings determine whether students can ask questions on their code in the corse, and if a draft answer is automatically generated. The draft answer is never shown directly to students, you decide as a teacher whether to use it or not.

Reviewer: ```suggestion
* `Questions`: these settings determine whether students can ask questions on their code in the course, and if a draft answer is automatically generated. The draft answer is never shown directly to students, you decide as a teacher whether to use it or not.
```
